src/paper/tasks.py

```diff
@@ -1308,107 +1307,38 @@ def pull_new_openalex_works(page=0, retry=0):
     except Exception as e:
         sentry.log_error(e)
 
+    total_papers_processed = 0
+    cursor = "*" # used for pagination
```

Comment:
Can we expand on this in a comment? I'm not sure how exactly this is used in pagination

Reply:
Done!